modules/sequenza/1.4/src/bash/filter_seqz.sh

```diff
@@ -18,11 +18,19 @@ set -euf -o pipefail
 zcat "${SEQZ_FILE}" \
 	| egrep -v "^chromosome" \
 	| awk 'BEGIN {FS="\t"} $9 == "het" {print $1 ":" $2}' \
-	| sort -S "${BUFFER_SIZE}" --parallel 10 \
+	| sort -S "${BUFFER_SIZE}" --parallel 8 \
```

Comment:
In theory the 8 should be a parameter that is passed in from Snakemake (ideally)